runtime/vm/createramclass.cpp

```diff
@@ -2257,12 +2257,10 @@ internalCreateRAMClassDone(J9VMThread *vmThread, J9ClassLoader *classLoader, J9C
 		if (NULL != superclass) {
 			/**
 			 * watched fields tag and exemption from validation are inherited from the superclass.
-			 * J9ClassHasIdentity is also inherited. If a class cannot be super class
-			 * of value types, its subclasses cannot be super of value types either.
```

Comment:
Please help me understand why `J9ClassHasIdentity` should no longer be inherited.

Reply:
In an older version of the spec java.lang.Object was considered to be neither value or identity type. In the current spec it is always considered an identity type, so it is not always correct to inherit this flag since a value class can have java.lang.Object has a superclass.
Since its easy to tell whether this flag should be set or not based on the ACC_IDENTITY flag I thought it made more sense to check for correct inheritance where J9ClassIsValueType is set.

Reply:
Thank you.